Backend/Lyzer/Controllers/ResultsController.cs

```diff
@@ -0,0 +1,27 @@
+using Lyzer.Common.DTO;
+using Lyzer.Services;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace Lyzer.Controllers
+{
+    [ApiController]
+    [Route("/api/v1/results")]
+    public class ResultsController : ControllerBase
+    {
+        private readonly ILogger<ResultsController> _logger;
+        private readonly ResultsService _resultsService;
+
+        public ResultsController(ILogger<ResultsController> logger, ResultsService resultsService)
+        {
+            _logger = logger;
+            _resultsService = resultsService;
+        }
+
+        [HttpGet("", Name = "GetLatestRaceResult")]
```

Comment:
This can also have a parameter of year, current can be a valid year.

Reply:
Hmm so far I do not think we should change this.

We are going to implement a completely different method for retrieving data for previous years in the future as that would be in the database, not Jolpica.

Instead of having to implement checks in the logic to see if it should be using cache or db, I would recommend completely separating concerns by using this endpoint for anything current/latest and then have a different endpoint for previous years data that is something more like `/api/v1/results/<year>`

Reply:
Perhaps, what I could do is change this to:

`/api/v1/results/current`

Reply:
Ok, so what I'm seeing is that you are wanting these endpoints to be always calling Jolpica, and caching the results as well. Then for previous years, they will be accessing the DB.

I think that was where my misunderstanding was. I definitely think `/api/v1/results/current` is better in this case, just to make it a little bit more verbose.

Reply:
Perfect yeah, let us chat a bit more on Wednesday.